guardian/support-frontend
# Validate zip code on checkout if not empty

## What are you doing in this PR?

Fix the zip code validation on non-generic checkout by including the "path" property, so it knows which field to attach the validation message to.



## How to test

In non-generic checkout 
- enter a zip code in personal details that doesn't match the pattern. Submit form. See error.
- leave zip code blank. Submit form. See no error.



## Screenshots

Before
![image](https://github.com/guardian/support-frontend/assets/114918544/41f4f449-7a4e-46fe-99ec-cacfb6d2a124)

After
![image](https://github.com/guardian/support-frontend/assets/114918544/9df579b6-8858-4197-987e-fde51abbc0b0)
